### DISCUSSION TITLE
Toast Documentation

### DISCUSSION THREAD

**Discussion by pguptaDS** [twilio-labs/paste#767] [Q&A] [answered]
Is there documentation on the toast component yet? I see it is now available in sketch but I wasn't sure about how to use it properly in my designs.

Thanks!

**Comment by SiTaggart** [comment, marked as answer]
It's being built right now. It should be merged in by Wednesday next week at the latest.

This is the preview, but there are feedback rounds still to go on it: https://paste-git-feat-toast.twilio-dsys.vercel.app/components/toast

**Comment by pguptaDS** [comment]
Got it. Thank you for sharing the preview, Simon! This helps a ton.
